jobs/uaa/spec

```diff
@@ -832,6 +832,9 @@ properties:
     default: true
   uaa.database.case_insensitive:
     description: "Set to true if you don't want to be using LOWER() SQL functions in search queries/filters, because you know that your DB is case insensitive. If this property is null, then it will be set to true if the UAA DB is MySQL and false otherwise, but even on MySQL you can override it by setting it explicitly to false"
+  uaa.database.test_while_idle:
```

Comment:
From reading the description you wrote below, and having no other idea what this thing does, it seems to me like the name should rather be `destroy_idle_connections` (to the database). 

Or am I misunderstanding what this is doing?

Reply:
This property is not about destroying all idle database connections, but about testing the idle connections (that are still kept in the connection pool) if they are still working. So only those connection who are not working anymore will be dropped out of the pool. If the connections are still ok, but idle, you still want to have them in the pool, as reusing them is faster than establishing a new connection. 

The name here matches the property name of the datasource and also the name that has to be used in the uaa.yml (only adjusted to using snake-case like the other properties also use that).

Reply:
OK, that makes sense. So then could the name be something like:
```cleanup_broken_connections```?

The fact that we are testing the connections is IMHO an implementation detail. The product consequence of this feature seems to be that broken connections are either kept in the pools forever, or removed once they are found.
Is that correct?